float64 m_timestamp

bool m_trackingSuccessful

ros_faceshift/fsQuaternionf m_headRotation
ros_faceshift/fsVector3f    m_headTranslation

float32 m_eyeGazeLeftPitch
float32 m_eyeGazeLeftYaw
float32 m_eyeGazeRightPitch
float32 m_eyeGazeRightYaw

float32[] m_coeffs

ros_faceshift/fsVector3f[] m_markers
